03_launch_mgmt_cluster.sh

```diff
@@ -276,6 +263,18 @@ function launch_cluster_api_provider_metal3() {
       kubectl scale -n metal3 deployment.v1.apps capm3-controller-manager --replicas 0
       nohup make run >> capm3.out.log 2>> capm3.err.log &
     fi
+
+    if [ "${BMO_RUN_LOCAL}" = true ]; then
+      touch bmo.out.log
+      touch bmo.err.log
+      kustomize build "$kustomize_overlay_path" | kubectl apply -f-
```

Comment:
why would you deploy BMO again ? it is deployed as part of CAPM3

Reply:
Oops, it was copy/pasted, fixed.